src/cpp/src/continuous_batching_impl.cpp

```diff
@@ -133,7 +133,7 @@ bool ContinuousBatchingPipeline::ContinuousBatchingImpl::has_non_finished_reques
     return !m_awaiting_requests.empty() || !m_requests.empty();
 }
 
-void ContinuousBatchingPipeline::ContinuousBatchingImpl::step() {
+size_t ContinuousBatchingPipeline::ContinuousBatchingImpl::step() {
```

Comment:
CB pipeline has a method, which returns PipelineMetrics
https://github.com/openvinotoolkit/openvino.genai/blob/29ea577a589319abbe63284bd9ae667fceba2042/src/cpp/include/openvino/genai/continuous_batching_pipeline.hpp#L26-L51

I think you can extend this structure with information you need (number of scheduled tokens) and call it after `step()` instead of patching return value with unexpected return type

Reply:
Done. Added a new field with number of generated tokens. No need to return it vis step() return value.